src/box/lua/schema.lua

```diff
@@ -1924,8 +1924,6 @@ local function check_ck_constraint_arg(ck_constraint, method)
 end
 box.internal.check_ck_constraint_arg = check_ck_constraint_arg
 
-box.internal.schema_version = builtin.box_schema_version
```

Comment:
Nit: You should also remove `box_schema_version` definition from the `ffi.cdef` above.

Reply:
Yeah, forgot about it, removed, thanks!